### PR TITLE
Time map/fast

### PR DESCRIPTION
**Pull request by EivydasKoc**
Add time-map/fast endpoint support as documented [here](https://docs.traveltime.com/api/reference/isochrones-fast)

Note: this endpoint does not support KML, we need to update the docs still.

### CODE REVIEW

**Review by MockusTravelTime — APPROVED** [review]

File: src/main/java/com/traveltime/sdk/dto/requests/TimeMapFastWktRequest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+@EqualsAndHashCode(callSuper = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TimeMapFastWktRequest extends TravelTimeRequest<TimeMapFastWktResponse> {
+    @NonNull
+    ArrivalSearches arrivalSearches;
+    @Builder.Default
+    boolean withHoles = true;
+
+    private AcceptType acceptType() {
+        if (withHoles) {
```
Comment (danielnaumau) [inline src/main/java/com/traveltime/sdk/dto/requests/TimeMapFastWktRequest.java:29]: return withHoles ? APPLICATION_WKT_JSON :  APPLICATION_WKT_NO_HOLES_JSON

Comment (danielnaumau) [reply src/main/java/com/traveltime/sdk/dto/requests/TimeMapFastWktRequest.java:29]: Use ternary operator here

File: src/main/java/com/traveltime/sdk/dto/requests/timemapfast/levelofdetail/CoarseGridLevelOfDetail.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Jacksonized
+@AllArgsConstructor
+@Value
+public class CoarseGridLevelOfDetail implements LevelOfDetail {
+  @NonNull
+  @Min(value = 600, message = "squareSize should be more than 600")
+  Integer squareSize;
```
Comment (danielnaumau) [inline src/main/java/com/traveltime/sdk/dto/requests/timemapfast/levelofdetail/CoarseGridLevelOfDetail.java:15]: we use 4 spaces everywhere, not 2

File: src/main/java/com/traveltime/sdk/dto/requests/timemapfast/levelofdetail/SimpleLevelOfDetail.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import lombok.NonNull;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Jacksonized
+@AllArgsConstructor
+@Value
+public class SimpleLevelOfDetail implements LevelOfDetail {
+  @NonNull
+  Level level;
```
Comment (danielnaumau) [inline src/main/java/com/traveltime/sdk/dto/requests/timemapfast/levelofdetail/SimpleLevelOfDetail.java:13]: 4 spaces

File: src/main/java/com/traveltime/sdk/dto/responses/timemapfast/ResponseProperties.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ResponseProperties {
```
Comment (danielnaumau) [inline src/main/java/com/traveltime/sdk/dto/responses/timemapfast/ResponseProperties.java:14]: should it be empty?

**Review by danielnaumau — CHANGES_REQUESTED** [review]

**Comment by danielnaumau** [comment]
Please, install this plugin https://plugins.jetbrains.com/plugin/7973-sonarlint and fix all style issues which it marks. Cause I didn't mention in the review all of them.

File: src/main/java/com/traveltime/sdk/dto/responses/timemapfast/ResponseProperties.java (hunk 4 above)
Comment (EivydasKoc) [reply src/main/java/com/traveltime/sdk/dto/responses/timemapfast/ResponseProperties.java:14]: This endpoint never returns any properties but I need a response type for geojson FeatureCollection<G, P> perhaps there is a better way of doing this is you could advise? If I would reuse existing one from current time-map we would jsut be adding needless nulls to response.

Comment (danielnaumau) [reply src/main/java/com/traveltime/sdk/dto/responses/timemapfast/ResponseProperties.java:14]: got it, so leave it like that

**Review by danielnaumau — APPROVED** [review]

**Review by EdvinasJusis — APPROVED** [review]

**Review by Donatas-L — APPROVED** [review]

**Pull request merged**